api/coreservice.go

```diff
@@ -257,10 +257,10 @@ func (core *coreService) Account(addr address.Address) (*iotextypes.AccountMeta,
 	if err != nil {
 		return nil, nil, status.Error(codes.NotFound, err.Error())
 	}
+	// TODO: deprecate nonce field in account meta
 	accountMeta := &iotextypes.AccountMeta{
 		Address:      addrStr,
 		Balance:      state.Balance.String(),
-		Nonce:        state.Nonce,
```

Comment:
let's deprecate in a further PR like the TODO said?

Reply:
it is part of this PR, we deprecate the concept of "confirmed nonce"

Reply:
further PR is to deprecate it in protobuf definition

Reply:
my worry is that this is breaking the API behavior, some user may be using `meta.Nonce` from returned value and this will break their code?